src/layout.css

```diff
@@ -46,48 +46,67 @@
  */
 
 /**
- * Grids are made by adding the `grid` class to a parent and `col col--{size}` classes to children. Column classes include `*-mm`, `*-ml`, and `*-mxl` variations to target screen sizes.
+ * Grids require at minimum two elements: A parent with the class `grid` and a child with the class `col`. By default, each `col` in a grid will occupy equal space. The `grid` class includes `*-mm`, `*-ml`, and `*-mxl` variations to target screen sizes.
  *
- * Grids and their columns use margin and padding for layout, so you should not apply any other margin or padding to these elements. Instead, apply such styling to parent and child elements, and use these elements for the grid layout only.
+ * When using these classes, do not apply left or right margin or padding directly to the \`grid\` or \`col\` elements or you will face unexpected side-effects.
  *
  * @group
  * @memberof Grid
  * @example
  * <div class='grid'>
- *   <div class='col col--2 border border--darken10'>col--2</div>
- *   <div class='col col--4 border border--darken10'>col--4</div>
- *   <div class='col col--6 border border--darken10'>col--6</div>
+ *   <div class='col border border--darken10'>one</div>
+ *   <div class='col border border--darken10'>two</div>
+ *   <div class='col border border--darken10'>three</div>
```

Comment:
It works well! I had to rewrite the col--{x} rules a bit to make it work though. There's a example of this in the col--{x} case.